[2025-04-10T09:58:46.423+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-10T09:58:46.512+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: et_extract_load.get_last_load manual__2025-04-10T09:58:41.902925+00:00 [queued]>
[2025-04-10T09:58:46.542+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: et_extract_load.get_last_load manual__2025-04-10T09:58:41.902925+00:00 [queued]>
[2025-04-10T09:58:46.543+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-04-10T09:58:46.549+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): get_last_load> on 2025-04-10 09:58:41.902925+00:00
[2025-04-10T09:58:46.554+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=27669) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-04-10T09:58:46.555+0000] {standard_task_runner.py:72} INFO - Started process 27670 to run task
[2025-04-10T09:58:46.556+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'et_extract_load', 'get_last_load', 'manual__2025-04-10T09:58:41.902925+00:00', '--job-id', '100', '--raw', '--subdir', 'DAGS_FOLDER/etl_extract_load.py', '--cfg-path', '/tmp/tmpounbf_ri']
[2025-04-10T09:58:46.556+0000] {standard_task_runner.py:105} INFO - Job 100: Subtask get_last_load
[2025-04-10T09:58:46.580+0000] {task_command.py:467} INFO - Running <TaskInstance: et_extract_load.get_last_load manual__2025-04-10T09:58:41.902925+00:00 [running]> on host 8bfd2e589ccf
[2025-04-10T09:58:46.619+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='et_extract_load' AIRFLOW_CTX_TASK_ID='get_last_load' AIRFLOW_CTX_EXECUTION_DATE='2025-04-10T09:58:41.902925+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-10T09:58:41.902925+00:00'
[2025-04-10T09:58:46.620+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-04-10T09:58:46.620+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-04-10T09:58:46.620+0000] {logging_mixin.py:190} INFO - Current task name:get_last_load state:running start_date:2025-04-10 09:58:46.537993+00:00
[2025-04-10T09:58:46.621+0000] {logging_mixin.py:190} INFO - Dag name:et_extract_load and current dag run status:running
[2025-04-10T09:58:46.621+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-04-10T09:58:46.626+0000] {base.py:84} INFO - Retrieving connection 'PostgresSQL_connection_1'
[2025-04-10T09:58:46.629+0000] {logging_mixin.py:190} INFO - Error: column "dim_product" does not exist
LINE 4:             WHERE table_name = dim_product;
                                       ^
[2025-04-10T09:58:46.629+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-04-10T09:58:46.634+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-04-10T09:58:46.635+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=et_extract_load, task_id=get_last_load, run_id=manual__2025-04-10T09:58:41.902925+00:00, execution_date=20250410T095841, start_date=20250410T095846, end_date=20250410T095846
[2025-04-10T09:58:46.646+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-04-10T09:58:46.646+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-04-10T09:58:46.647+0000] {logging_mixin.py:190} INFO - Dag name:et_extract_load queued_at:2025-04-10 09:58:41.912089+00:00
[2025-04-10T09:58:46.647+0000] {logging_mixin.py:190} INFO - Task hostname:8bfd2e589ccf operator:PythonOperator
[2025-04-10T09:58:46.690+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-10T09:58:46.704+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-04-10T09:58:46.715+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
